drupal/sites/default/boinc/themes/boinc/templates/views-view-table--boinc-team-forum-topics.tpl.php

```diff
@@ -51,29 +50,29 @@
       if ($topic_index == 1) {
         $row_class .= ' first';
       }
-      if ($topic->sticky=="True") {
+      if ($result[$id]->node_boincteam_forum_node_sticky) {
         $row_class .= ' sticky';
       }
-      elseif (!$first_non_sticky AND !($topic->sticky=="True")) {
+      elseif (!$first_non_sticky AND !($result[$id]->node_boincteam_forum_node_sticky)) {
         $row_class .= ' first-non-sticky';
         $first_non_sticky = TRUE;
       }
       if ($topic_index == $topic_count) {
         $row_class .= ' last';
       }
-      if ($topic->timestamp=="new" OR $topic->new_comments) {
+      if ($topic->timestamp OR $topic->new_comments) {
```

Comment:
As I expected. The name is misleading. I would suggest to use `empty()` or `isset()` in this case to avoid potential E_NOTICE warnings when the variable is not set. If it is guaranteed to always contain an empty string "" or "new" then use `!empty($topic->timestamp)`